tools/flang2/flang2exe/cgmain.cpp

```diff
@@ -1459,6 +1493,15 @@ schedule(void)
     for (ilt = BIH_ILTFIRST(bih); ilt; ilt = ILT_NEXT(ilt))
       build_csed_list(ILT_ILIP(ilt));
 
+  /* Process variables with parameter attribute to generate debug info, if
+     debug is on. */
+  if (!XBIT(49, 0x10) && flg.debug
```

Comment:
Could you document this use of `XBIT(49, 0x10)` in `tools/flang2/docs/xflag.n`?

Reply:
This is an existing option you can find other usage of it in files semant.c and semant2.c, which was probably earlier introduced to guard (possibly unfinished) changes.
the current condition is to make sure that experimental (and undocumented) option is not mentioned. if user is not aware of that option this condition will always pass.

Reply:
My understanding of xflags is that they should already be considered internal options. Unlike driver options, they are not designed to be used by end users. Not having a definitive list of xflags and a version-controlled way to reserve bits makes it very confusing for developers, IMHO. I think we should discuss this "current condition" but it is not a problem that should block this pull request.